target/linux/generic/config-6.6

```diff
@@ -324,7 +324,9 @@ CONFIG_ARCH_MMAP_RND_COMPAT_BITS_MIN=8
 # CONFIG_ARM64_64K_PAGES is not set
 # CONFIG_ARM64_AMU_EXTN is not set
 # CONFIG_ARM64_BTI is not set
+CONFIG_ARM64_CNP=y
 # CONFIG_ARM64_E0PD is not set
+CONFIG_ARM64_EPAN=y
```

Comment:
This is supported starting with ARMv8.7, I think we can disable it by default.

Reply:
I agree, we won't be seeing ARMv8.7 anytime soon in OpenWrt.